.github/workflows/build.yml

```diff
@@ -46,11 +46,14 @@ jobs:
         # Build in release without `testing` feature, this should work without `hotshot_example` config.
         run: |
           cargo build --locked --release --workspace
+      
+      - name: Build sequencer-sqlite 
+        run: cargo build --locked --release --manifest-path ./sequencer-sqlite/Cargo.toml --target-dir ./target
 
       - name: Build Espresso Dev Node
         # Espresso Dev Node currently requires testing feature, so it is built separately.
         run: |
-          cargo build --locked --release --features testing --bin espresso-dev-node
+          cargo build --locked --release --features "testing embedded-db" --bin espresso-dev-node
```

Comment:
Can we explicitly specify the `embedded-db` features everywhere? I think in other places we are just specifying `--all-features` which isn't clear. Especially since we were using `--all-features` previously (*I think*).

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/2284/commits/09e08788d0af92129c96c235ebb498053a9aa16a